apps/client/src/components/atoms/text/text.css

```diff
@@ -0,0 +1,35 @@
+.font-major-mono-display {
+  font-family: 'Major Mono Display', monospace;
+}
+
+.font-roboto-mono {
+  font-family: 'Roboto Mono', monospace;
+}
+
+.font-sora {
+  font-family: 'Sora', sans-serif;
+}
+
+.variant-display {
```

Comment:
Why are you using CSS to get this done? Please use tailwind classes

Reply:
Removed 60053a515725709ad13b0ffa3347a696845ad0b8